docs/UserGuide.md

```diff
@@ -162,6 +175,19 @@ Examples:
 * `edit 2 g/f`
 * `edit 3 g/F`
 
+__Optional Parameter 2: Date of Birth__
+
+Format: `edit INDEX [d/DOB]`
+
+- Edits the date of birth of a person in the contact list. Date formats accepted are: d.M.yyyy, dd MMM yyyy, M d, yyyy
```

Comment:
Ah it needs the ; separator here as well for the 3 types of formats

Reply:
ok added! changed the date formats as previously stated as well sry forgot to add that in